cypress/integration/eholdings/eholdings-titles-packages.spec.js

```diff
@@ -0,0 +1,39 @@
+import eHoldingSearch from '../../support/fragments/eholdings/eHoldingsSearch';
+import eHoldingsTitlesSearch from '../../support/fragments/eholdings/eHoldingsTitlesSearch';
+import testTypes from '../../support/dictionary/testTypes';
+import features from '../../support/dictionary/features';
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import users from '../../support/fragments/users/users';
+
+describe('ui-eholdings: Search titles', () => {
+  const testData = {
+    title: 'Chemical Engineering',
+    publicationType: 'Journal'
+  };
+
+  before('Preconditions', () => {
+    cy.createTempUser([permissions.uieHoldingsRecordsEdit.gui,
+      permissions.uieHoldingsPackageTitleSelectUnselect.gui]).then(userProperties => {
+      testData.userId = userProperties.userId;
+      cy.login(userProperties.username, userProperties.password);
+      cy.visit(TopMenu.eholdingsPath);
```

Comment:
Can you move the path parameter from the cy.visit function to the cy.login function and not use cy.visit function?

Reply:
yep